.github/workflows/deleted-files.yml

```
@@ -0,0 +1,38 @@
+name: Deleted files
+
+on:
+  pull_request:
+    branches:
+      master
+
+jobs:
+  list-deleted-files:
+    runs-on: ubuntu-latest
+    name: Detect deleted files
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Get changed docs md files
+        id: changed-files
+        uses: tj-actions/changed-files@v37
+        with:
+          files: docs/**/*.md
+      - name: Format list of deleted files
+        id: format-output
+        env:
+          ORIGINAL: '${{ steps.changed-files.outputs.deleted_files }}'
+        run: |
+          echo "FORMATTED=$(echo $ORIGINAL | sed 's/ /`<br>`/g')" >> $GITHUB_OUTPUT
```

I tried different flavors of newlines here with little success, such as:

```suggestion
          echo "FORMATTED=$(echo $ORIGINAL | sed 's/ /\\n/g')" >> $GITHUB_OUTPUT
```

---

Try this instead:

```suggestion
          echo "FORMATTED=$(echo $ORIGINAL | sed 's/ /`  <br />`/g')" >> $GITHUB_OUTPUT
```

I'm thinking that GitHub's Markdown renderer appends the content of multiple lines together on the same "HTML line". According to the Markdown specification, we actually [need to add two spaces at the end of the lines](https://www.markdownguide.org/basic-syntax#line-breaks) to force them to be rendered with line breaks.

---

Correct, there's no need to change this. My suggestion is only an example.
What I was trying to achieve was printing all files as a unique code block:
```
docs/faq/code-analysis/does-codacy-check-for-dependencies.md
docs/getting-started/adding-a-codacy-badge.md
docs/index.md
```

But this would not work, generating something like:
```
docs/faq/code-analysis/does-codacy-check-for-dependencies.md\ndocs/getting-started/adding-a-codacy-badge.md\ndocs/index.md
```

This is why I was trying a newline char instead of its HTML equivalent...but code blocks escape the newline char, too. Using any kind of literal newline would either break the pipeline or not display properly.

I consider this a cosmetic improvement though, and not worth banging our heads too much for.

---

Ahh, now I understand. But I agree, it's not worthwhile chasing this minor cosmetic improvement. :+1: